configure.ac

```diff
@@ -97,7 +97,7 @@ AC_ARG_ENABLE(pipewire,
 	      [AS_HELP_STRING([--enable-pipewire],[Enable PipeWire support. Needed for screen cast portal])],
 	      enable_pipewire=$enableval, enable_pipewire=yes)
 if test x$enable_pipewire = xyes ; then
-	PKG_CHECK_MODULES(PIPEWIRE, [libpipewire-0.2 >= 0.2.6])
+	PKG_CHECK_MODULES(PIPEWIRE, [libpipewire-0.3 >= 0.2.9])
```

Comment:
Maybe it should be >= 0.2.90? I don't know what are your plans with PW 0.2, but latest release is 0.2.7 so if you decide to do two more stable releases, then this check will pass and it won't build.